cc3d/core/PySteppables.py

```diff
@@ -1891,6 +1891,14 @@ def get_cell_boundary_pixel_list(self, cell, neighbor_order=-1):
     def moveCell(self, cell, shiftVector):
         return self.move_cell(cell=cell, shift_vector=shiftVector)
 
+    def point3d_to_tuple(self, pt: CompuCell.Point3D) -> tuple:
```

Comment:
Could we add this instead to the python definition for Point3D, Dim3D, etc.? Something like "to_tuple()". I feel like that would be much more generally useful and also save on a little additional clutter for this class definition.